src/tersets.zig

```diff
@@ -38,7 +38,7 @@ pub const Method = enum {
 };
 
 /// A point represented by `time` and `value`.
-pub const Point = struct { time: usize, value: f64 };
+pub const Point = struct { time: f64, value: f64 };
```

Comment:
Maybe make `Point` and `Segment` generic like [`Array`](https://github.com/cmcuza/TerseTS/blob/4c4dc15a68fd1146377864e9b4e12779329f6c83/src/capi.zig#L96) so the different compression methods can use either discrete or continuous time depending on their need.

Reply:
Ummm that is a better idea.